src/test/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintServiceTest.java

```diff
@@ -54,22 +53,18 @@ public void shouldSendLetterWithDocumentsAsInGivenOrder() {
         //given
         String authValue = "AuthValue";
         when(authTokenGenerator.generate()).thenReturn(authValue);
-        Map<String, Object> pinContents = new HashMap<>();
-        Document defendantLetterDocument = new Document("pinTemplate", pinContents);
-        Map<String, Object> claimContents = new HashMap<>();
-        Document sealedClaimDocument = new Document("sealedClaimTemplate", claimContents);
         Claim claim = SampleClaim.getDefault();
-
         Map<String, Object> additionalData = new HashMap<>();
         additionalData.put(ADDITIONAL_DATA_LETTER_TYPE_KEY, ADDITIONAL_DATA_LETTER_TYPE_VALUE);
         additionalData.put(ADDITIONAL_DATA_CASE_IDENTIFIER_KEY, claim.getId());
         additionalData.put(ADDITIONAL_DATA_CASE_REFERENCE_NUMBER_KEY, claim.getReferenceNumber());
-
-        DocumentReadyToPrintEvent event
-            = new DocumentReadyToPrintEvent(claim, defendantLetterDocument, sealedClaimDocument);
+        Map<String, Object> pinContents = new HashMap<>();
+        Document defendantLetterDocument = new Document("pinTemplate", pinContents);
+        Map<String, Object> claimContents = new HashMap<>();
```

Comment:
Can be inlined

Reply:
I thought of same, however left extracted just to know what is this empty hash map